### PR TITLE
Add console nginx image

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: console-nginx/Dockerfile [hunk 1]
```diff
@@ -0,0 +1 @@
+FROM nginxinc/nginx-unprivileged:1.19.6-alpine
```
Comment (a-thaler) [inline console-nginx/Dockerfile:1]: What's the point of having that image re-builded here? Why not just replicating the image directly with the image-syncer

Comment (parostatkiem) [reply console-nginx/Dockerfile:1]: It's just a base to start work with. I'm going to modify and extend the image.

Comment (a-thaler) [reply console-nginx/Dockerfile:1]: So this PR in it's current shape brings no value value, please do not merge it till there is a substantle improvement visible ;)

btw, if you re-build the image anyway, then you should think of making it based on alpine in it's latest version and copy over the binaries. That will assure that latest packages are getting used with less security vulnerabilities in the base image. And you have the chance at any time to update the base image version independent of the upstream nginx release cycle. That might be a first valuable step to start with.

Comment (parostatkiem) [reply console-nginx/Dockerfile]: Changed

File: console-nginx/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.13.1
+
+LABEL source git@github.com:kyma-project/console.git
+
+RUN apk add --update nginx=1.18.0-r13 && rm -rf /var/cache/apk/*
```
Comment (a-thaler) [inline console-nginx/Dockerfile:5]:
```suggestion
apk --no-cache upgrade &&\
    apk --no-cache add nginx=1.18.0-r13
```

**Review by a-thaler — APPROVED** [review]

**Review by majakurcius — APPROVED** [review]

**Pull request merged**